src/core/ext/filters/client_channel/subchannel_stream_client.cc

```diff
@@ -124,40 +127,39 @@ void SubchannelStreamClient::StartRetryTimerLocked() {
   if (event_handler_ != nullptr) {
     event_handler_->OnRetryTimerStartLocked(this);
   }
-  Timestamp next_try = retry_backoff_.NextAttemptTime();
+  const Timestamp next_try = retry_backoff_.NextAttemptTime();
+  const Duration timeout = next_try - Timestamp::Now();
```

Comment:
nit: this could be one line, such as `  const Duration timeout = retry_backoff_.NextAttemptTime() - Timestamp::Now();`.

Reply:
good point, done.